src/PermanentFilterProvider.tsx

```diff
@@ -1,29 +1,37 @@
 import { noop } from 'lodash';
 import React, { createContext, useCallback, useEffect, useReducer } from 'react';
 
-import { readFromStore } from './helpers';
+import { readFromStore, storageHelper } from './helpers';
 import {
   DATA_PROVIDER_FILTER_KEY,
+  FilterAction,
+  FilterReducerAction,
   MOWAS_REGIONAL_KEYS,
   MowasFilterAction,
   MowasFilterReducerAction,
   mowasRegionalKeysReducer,
+  OverlayFilterAction,
+  overlayFilterReducer,
+  OverlayFilterReducerAction,
   permanentFilterReducer
 } from './reducers';
-import { FilterAction, FilterReducerAction } from './types';
 
 type PermanentFilterProviderValues = {
   dataProviderDispatch: React.Dispatch<FilterReducerAction>;
   dataProviderState: string[];
   mowasRegionalKeysDispatch: React.Dispatch<MowasFilterReducerAction>;
   mowasRegionalKeysState: string[];
+  filterDispatch: React.Dispatch<OverlayFilterReducerAction>;
+  filterState: string[];
```

Comment:
resource...

Reply:
name change implemented with this commit
https://github.com/smart-village-solutions/smart-village-app-app/pull/1188/commits/6832d3814e0fced2f8c3613d1e569788bdd5398a